### PR TITLE
chore(agent): remove NodeId from config

### PR DESCRIPTION
**Pull request by polaroi8d**
Co-authored-by: Ritvik Singh <hackerer528@gmail.com>

### CODE REVIEW

File: golang/internal/config/config.go [hunk 1]
```diff
@@ -21,12 +21,11 @@ type CommonConfiguration struct {
 	Debug                bool          `yaml:"debug"                 env:"DEBUG"                   env-default:"false"`
 	ImportContainerImage string        `yaml:"importContainerImage"  env:"IMPORT_CONTAINER_IMAGE"  env-default:"rclone/rclone:1.57.0"`
 	IngressRootDomain    string        `yaml:"ingressRootDomain"     env:"INGRESS_ROOT_DOMAIN"     env-default:""`
-	// TODO(c3ppc3pp): custom UUIDv4 setter
-	NodeID            string        `yaml:"nodeID" env:"NODE_ID" env-default:"cb7e9573-9a43-4d5b-8005-eb8bb7a423c4"`
-	ReadHeaderTimeout time.Duration `yaml:"readHeaderTimeout"    env:"READ_HEADER_TIMEOUT"      env-default:"15s"`
-	Registry          string        `yaml:"registry"             env:"REGISTRY"                 env-default:"index.docker.io"`
-	RegistryPassword  string        `yaml:"registryPassword"     env:"REGISTRY_PASSWORD"        env-default:""`
-	RegistryUsername  string        `yaml:"registryUsername"     env:"REGISTRY_USERNAME"        env-default:""`
+	NodeID               UUID          `yaml:"nodeID" env:"NODE_ID" env-default:"cb7e9573-9a43-4d5b-8005-eb8bb7a423c4"`
```
Comment (nandor-magyar) [inline golang/internal/config/config.go:24]: Align it with the other lines please.

**Review by nandor-magyar — APPROVED** [review]
Missing indent, otherwise ok :ok_hand:

File: golang/internal/config/config.go (hunk 1 above)
Comment (polaroi8d) [reply golang/internal/config/config.go:24]: Why linter doesn't show errors to me?

Comment (polaroi8d) [reply golang/internal/config/config.go]: Updated.

**Review by balanceee — DISMISSED** [review]

**Review by balanceee — APPROVED** [review]

**Review by robot9706 — DISMISSED** [review]
LGTM 🍿

**Pull request merged**